Administrator-Task y errores de Task arreglados.

alejandropiuryp opened the pull request:
Se ha creado la funcionalidad misma implementada para el rol de Manager ahora también para el rol de Administrator. Además se han arreglado errores de la entidad Task y solucionadas las brechas de seguridad que existían forzando los formularios relacionados con la entidad "Task".

alejandropiuryp committed:
Creación de una Task: datos vacíos fixed

Se ha arreglado el formulario de creacción de una tarea con los datos vacíos

alejandropiuryp committed:
Conflictos de Task Validator arreglado

alejandropiuryp committed:
Conflictos con Task validator arreglados

alejandropiuryp committed:
Task-Administrator y errores varios fixed

Se han arreglado errores de la entidad Task y del modelo UML así como brechas de seguridad en los formularios de Task y se ha añadido la funcionalidad completa para el rol de Administrator.

manpercar1 reviewed (changes requested):
Al crear tasks tanto con rol de manager como con rol de administrador, se obtiene error de "No autorizado"

alejandropiuryp committed:
Create Task - Administrator y Manager fixed

Arreglado el pequeño error que impedía crear una tarea por permisos de autorización

manpercar1 reviewed (approved):
Todo correcto

manpercar1 committed:
Merge branch 'develop' into feature/Managers-Task
